lib/web_ui/dev/test_platform.dart

```diff
@@ -142,9 +142,9 @@ class BrowserPlatform extends PlatformPlugin {
   }
 
   Future<shelf.Response> _screeshotHandler(shelf.Request request) async {
-    if (browserName != 'chrome') {
-      throw Exception('Screenshots tests are only available in Chrome.');
-    }
+    // if (browserName != 'chrome') {
+    //   throw Exception('Screenshots tests are only available in Chrome.');
+    // }
```

Comment:
Revert?

Reply:
thanks! they shouldn't even be in this branch puffff :D